tests/pipeline/test_pipeline_with_transcoding.py

```diff
@@ -274,6 +274,30 @@ class TestComplexPipelineWithTranscoding:
 
     """
 
+    def test_from_to_nodes_transcoded_names(self, complex_pipeline):
+        """New pipelines contain all nodes that depend on node8, node7."""
+        from_nodes_pipeline = complex_pipeline.from_nodes("node8")
+        nodes = {node.name for node in from_nodes_pipeline.nodes}
+
+        assert len(from_nodes_pipeline.nodes) == 9
+        assert nodes == {
+            "node1",
+            "node2",
+            "node3",
+            "node4",
+            "node5",
+            "node6",
+            "node7",
+            "node8",
+            "node10",
+        }
+
+        to_nodes_pipeline = complex_pipeline.to_nodes("node7")
+        nodes = {node.name for node in to_nodes_pipeline.nodes}
+
+        assert len(to_nodes_pipeline.nodes) == 3
+        assert nodes == {"node7", "node8", "node9"}
+
```

Comment:
I would break this down into 2 unit tests, for separation of concerns.

Reply:
I actually tried that first but it would bump into pylint's limit of 20 public methods per class and I didn't want to override that limit.

Reply:
Ah I see, in that case you can stick a 
```
# pylint: disable=too-many-public-methods
```
We use that in `test_context` already, and it's just test code so pylint is being a bit dramatic. What do you think?

Reply:
Agree, done in 37a381b.